crates/core_simd/src/ops.rs

```diff
@@ -367,6 +394,10 @@ macro_rules! impl_unsigned_int_ops {
                             if rhs == 0 {
                                 panic!("attempt to calculate the remainder with a divisor of zero");
                             }
+                            if <$scalar>::MIN != 0  && rhs == -1 as _ {
+                                panic!("rem(MIN, -1) is undefined");
```

Comment:
The panic message should use a consistent formatting between the division and remainder cases, and ideally between the scalar and vector cases as well, so either here it should be
```rust
                                 panic!("MIN modulo -1 is undefined"); 
```

or the `fn div` case should use the `div(MIN, -1)` notation, so back there it would be
```rust
                                 panic!("div(MIN, -1) is undefined"); 
```

but in any case, the actual message here is
```rust
thread 'main' panicked at 'attempt to divide with overflow', src/main.rs:3:17
```

But honestly I can be persuaded on any messaging as long as it's consistent.

Reply:
Sure, I'm going with the first on option you suggested.